packages/config-array/src/config-array.js

```diff
@@ -606,7 +606,7 @@ export class ConfigArray extends Array {
 		 * @property basePath
 		 * @type {string}
 		 */
-		this.basePath = basePath || "/";
+		this.basePath = basePath;
```

Comment:
Can we validate here so we can throw a more specific error, like "basePath must be a non-empty string"?

Reply:
Done in fb712b13ff3a15fbab0117abb858b147ec9e39ff, thanks.